### ISSUE TITLE
Demo app?

### ISSUE THREAD

**Issue by tcies** [ethz-asl/map_api#2]
Please let us know in comments if you would be interested in a simple Map API example app. Note that in the meantime the application of most functions can be gleaned from the unit tests:
https://github.com/ethz-asl/map_api/tree/master/map-api/test

**Comment by tongbaiming** [comment]
Yes, I am very interested in a simple MAP API  example app. And the more example the better.
